packages/server/test/integration/client-tokens.test.ts

```diff
@@ -0,0 +1,140 @@
+import { validate as validateUuid } from 'uuid'
+import { ClientTokenService } from '../../src/client-tokens/service'
+import { ClientToken } from '../../src/client-tokens/types'
+import { Client } from '../../src/clients/types'
+import { Provider } from '../../src/providers/types'
+import { app, randStr, setupApp } from './utils'
+
+describe('ClientTokens', () => {
+  let clientTokens: ClientTokenService
+  let querySpy: jest.SpyInstance
+  let state: { provider: Provider; client: Client; rawToken?: string; clientToken?: ClientToken }
+
+  beforeAll(async () => {
+    // This should be reset after those tests
+    process.env.ENABLE_EXPERIMENTAL_SOCKETS = '1'
```

Comment:
I don't think there is anything wrong with enabling this feature across all integration tests.